action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Altostra Login'
+description: 'GitHub Action to login against an Altostra account'
+author: 'Altostra'
+branding:
+  icon: 'triangle'
+  color: 'purple'
```

Comment:
What does this mean?

Reply:
The little signs at the side of the action. Still haven't got it to work , but it's per their documentation :)